dbt/adapters/redshift/relation_configs/materialized_view.py

```diff
@@ -199,11 +206,12 @@ def parse_relation_results(cls, relation_results: RelationResults) -> Dict:
                 {"dist": RedshiftDistConfig.parse_relation_results(materialized_view)}
             )
 
-        # TODO: this only shows the first column in the sort key
-        if materialized_view.get("sortkey1"):
-            config_dict.update(
-                {"sort": RedshiftSortConfig.parse_relation_results(materialized_view)}
-            )
+        if columns:= relation_results.get("columns"):
+            sort_columns = [row for row in columns.rows if row.get("sort_key_position") > 0]
```

Comment:
In fact, if a column is not part of the sort_key, it is returned as 0.
<img width="580" alt="image" src="https://github.com/user-attachments/assets/6331867f-a087-4a11-b6bd-e484c069a7ac">

But I think it's fine to default the `.get()` with 0

Reply:
Oh great! I agree that we should still default. Sometimes data platforms update metadata defaults like this because it's rarely used by end users. This would protect us a bit if that happens.